ludwig/combiners/combiners.py

```diff
@@ -41,23 +42,65 @@
 logger = logging.getLogger(__name__)
 
 
-class ConcatCombiner(LudwigModule):
+# super class to house common properties
+class CombinerClass(LudwigModule):
+    @property
+    def effective_input_shape(self) -> torch.Size:
```

Comment:
it looks to me that this function is relevant only for sequence and sequenceconcat combiners.
Maybe we can all this `concatenated_size` and leave it abstract and each combiner implement it the way they want.
This implementation would work for the seq ones, the current `_input_last_dimension_size` would work cor the concat one.
What do you think?

Reply:
Sounds reasonable.  I'll give this a try.

Reply:
done.